tests/ui/methods/disambiguate-multiple-blanket-impl.stderr

```diff
@@ -29,10 +29,10 @@ LL |     fn foo(&self) {}
    |     ^^^^^^^^^^^^^
 help: use fully-qualified syntax to disambiguate
    |
-LL |     <T as A>::foo(&s);
-   |     ~~~~~~~~~~
-LL |     <T as B>::foo(&s);
-   |     ~~~~~~~~~~
+LL |     A::foo(&s);
```

Comment:
This is more correct, since we're 1. not suggesting a random type parameter and 2. eliding an unnecessary `Self` param.